pubtools/_quay/operator_pusher.py

```diff
@@ -229,6 +229,12 @@ def pubtools_iib_get_common_args(cls, target_settings):
         Returns (([str]), {str:str}):
             Tuple of arguments and environment variables to be used when calling pubtools-iib.
         """
+        settings = target_settings.copy()
+        for key in ["iib_overwrite_from_index_token"]:
```

Comment:
What? Why is this iterated?

Reply:
I don't think this accomplishes what you want it to accomplish

Reply:
I put it in the list as in the future there could be more override settings. Also we don't want to allow to overwrite any arbitrary settings

Reply:
This seems really strange to me. You give the user the option to override target settings, but in reality it's not going to work (except for one setting). The settings are already passed as a parameter, isn't it much easier and clearer to just modify them as desired before calling this method?

Reply:
I made it clear that I disagree with this approach, but shouldn't you also add iib_overwrite_from_index to this list?